teamscale_precommit_client/precommit_client.py

```diff
@@ -67,6 +73,11 @@ def print_precommit_results_as_error_string(self, include_findings_in_changed_co
             for formatted_finding in self._format_findings(findings_in_changed_code):
                 print(formatted_finding)
 
+        red_added_findings = list(filter(lambda finding: finding.assessment == "RED", added_findings))
+        return len(red_added_findings) > 0
+
+    def print_other_findings_as_error_string(self, include_existing_findings=True, include_all_findings=True):
```

Comment:
How about only providing include_all_findings as a parameter? You already check in the run method if any of both parameters is set and do not call the method otherwise. Then within this method we could simply adjust the uniform_path if include_all_findings is set to True and get rid of the other if in line 85.

Reply:
Nice catch, I didn't notice there was redundancy there. Thanks.